app/views/hyrax/embargoes/_list_active_embargoes.html.erb

```diff
@@ -0,0 +1,16 @@
+<table class="embargoes table">
+  <tbody>
+  <tr>
+    <th>Type of Work</th><th>Title</th><th>Current Visibility</th><th>Embargo Release Date</th><th>Visibility will Change to</th>
```

Comment:
Agree, @cbeer -- I was thinking we could ask @darrenleeweber to take that on immediately following this PR so we wouldn't have to do said diving.